infra/utils/sw-env.js

```diff
@@ -18,6 +18,17 @@ const swEnv = makeServiceWorkerEnv();
 // This is needed to ensure new URL('/', location), works.
 swEnv.location = 'https://example.com';
 
+// See https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/FetchEvent
```

Comment:
I know @philipwalton bumped into this as well, but while the `FetchEvent` is mocked in https://github.com/pinterest/service-workers/blob/master/packages/service-worker-mock/utils/events.js it isn't exported from there, and I needed it to be defined in the global scope in order to properly run the tests.

If this isn't the best practice for getting it defined globally, I'm happy to adjust.

Reply:
This seems like it should be handled by the mock lib. Just put this up for you. Thoughts?
[https://github.com/pinterest/service-workers/pull/50](https://github.com/pinterest/service-workers/pull/50)